src/main/java/vg/civcraft/mc/citadel/command/Acid.java

```diff
@@ -105,4 +110,17 @@ public boolean execute(CommandSender sender, String[] args) {
 	public List<String> tabComplete(CommandSender sender, String[] args) {
 		return new ArrayList<>();
 	}
+
+	public boolean isChestBeingAcidBlocked (Block block) {
+		if (block.getType() != Material.CHEST || block.getType() != Material.TRAPPED_CHEST) {
```

Comment:
Should use some other check to see if its an instance of container or something like that, not just those two types of chests